gnuradio-runtime/python/gnuradio/gr/bindings/logger_python.cc

```diff
@@ -14,152 +15,127 @@
 /* BINDTOOL_GEN_AUTOMATIC(0)                                                       */
 /* BINDTOOL_USE_PYGCCXML(0)                                                        */
 /* BINDTOOL_HEADER_FILE(logger.h)                                        */
-/* BINDTOOL_HEADER_FILE_HASH(6d4f118d476b888f79737f835ed27a4d)                     */
+/* BINDTOOL_HEADER_FILE_HASH(7f3cbb9463e52829b27c58d9dd41b422)                     */
 /***********************************************************************************/
 
 #include <pybind11/complex.h>
 #include <pybind11/pybind11.h>
 #include <pybind11/stl.h>
+#include <spdlog/common.h>
 
 namespace py = pybind11;
 
 #include <gnuradio/logger.h>
+#include <logger_pydoc.h>
 
 void bind_logger(py::module& m)
 {
-    using logger = gr::logger;
-    using logger_config = gr::logger_config;
+    py::enum_<spdlog::level::level_enum>(m, "level_enum")
```

Comment:
This is a very small nit, but the terminology "_enum" isn't used anywhere else.  Perhaps this should be "log_levels", so that in python it shows up as
mylogger.set_level(gr.log_levels.info)